Model/Config/ModuleConfig.php

```diff
@@ -189,7 +189,14 @@ public function canProcessRoute($destinationCountryId, $store = null)
      */
     public function canProcessMethod($shippingMethod, $store = null)
     {
-        return in_array($shippingMethod, $this->getShippingMethods($store));
+        $configuredMethods = $this->getShippingMethods($store);
+        foreach ($configuredMethods as $method) {
+            if (strpos($shippingMethod, $method)) {
```

Comment:
`strpos($shippingMethod, $method)` can return 0 as a valid result, which will result in a false negative. It's better to do a type save check here `strpos($shippingMethod, $method) !== false`.